android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -799,6 +799,26 @@ public void getCurrentPosition(Promise promise) {
         }
     }
 
+    public void getCurrentPlaybackStatus(Promise promise) {
+        if (player != null) {
+            int playbackState = player.getPlaybackState();
+            switch (playbackState){
+                case Player.STATE_IDLE -> promise.resolve("idle");
```

Comment:
I think you should add breaks in the switch case ?!

Reply:
It doesn't need but I added just to be on the safe side.